### PR TITLE
Add contributing info file

### LINKED ISSUE #110: Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tparchambault**
Closes #110

### CODE REVIEW

**Comment by tparchambault** [comment]
@jw3 , do you have any thoughts re: Code of Conduct? I just stole most of this text and modified for our project. Let me know if you have any specifics to include.

**Comment by jw3** [comment]
Might also add a Getting Help section and point to the Discussions section on the GitHub project

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+which under the hood is executing the following with the `develop` argument to setup.py:
+
+```{shell}
+pipenv install --dev
+pipenv shell
+python3 setup.py [develop | install]
+```
+### Using the make command driver
```
Comment (jw3) [inline CONTRIBUTING.md:32]: Move this to the wiki

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+pip3 install --user pipenv
+```
+
+To build and install the bindings run the following from the top level directory of this repository:
+
+```{shell}
+make shell
+```
+
+which under the hood is executing the following with the `develop` argument to setup.py:
```
Comment (jw3) [inline CONTRIBUTING.md:25]: I think cut out this bit about what happens under the hood

Comment (tparchambault) [reply CONTRIBUTING.md:25]: Done

File: CONTRIBUTING.md (hunk 1 above)
Comment (tparchambault) [reply CONTRIBUTING.md:32]: Done

**Comment by jw3** [comment]
Added #386 after thinking about the last section in the readme and how to determine whether a user should open an issue or start a discussion.  Doesn't need added in this PR.

**Pull request merged**